burn-compute/src/tune/operation.rs

```diff
@@ -16,6 +28,12 @@ pub trait AutotuneOperationSet<K>: Send {
     /// Returns the operation for the given index, matching the order
     /// returned by autotunables. Operation obtained here runs on original tensors
     fn fastest(self: Box<Self>, fastest_index: usize) -> Box<dyn AutotuneOperation>;
+
+    /// Compute checksum for the set by concatenating the op names together
```

Comment:
The doc is wrong, the implementations are free to implement this function with another strategy. => `Compute a checksum that can invalidate outdated cached auto-tune results.`